falcon/testing/helpers.py

```diff
@@ -734,12 +748,16 @@ async def _collect(self, event: Dict[str, Any]):
                     #   different raised error types or to pass through a
                     #   raised HTTPError status code.
                     self._close_code = desired_code
+                    self._close_reason = reason
                 else:
                     # NOTE(kgriffs): Force the close code to this since it is
                     #   similar to what happens with a real web server (the HTTP
                     #   connection is closed with a 403 and there is no websocket
                     #   close code).
                     self._close_code = WSCloseCode.FORBIDDEN
+                    self._close_reason = falcon.util.code_to_http_status(
```

Comment:
As I understand, this would include the preceding integer status code as part of reason.
I don't think we want this, at least it is not in line with the definition of the default reasons.